client/my-sites/site-settings/date-time-format/index.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import HeaderCake from 'components/header-cake';
+import { getSelectedSite } from 'state/ui/selectors';
+
+export const DateTimeFormat = ( { translate, site } ) => {
+	const goBack = () => {
+		page( '/settings/general/' + site.slug );
+	};
+
+	return (
+		<div className="main main-column date-time-format" role="main">
+			<DocumentHead title={ translate( 'Manage Date and Time Format Configurations' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Date and time format' )` *ES Score: 13.17*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/site-settings/date-format-sub-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).